chainerrl/q_functions/state_action_q_functions.py

```diff
@@ -130,25 +148,34 @@ def __call__(self, state, action):
 
 class FCBNLateActionSAQFunction(chainer.Chain, StateActionQFunction,
                                 RecurrentChainMixin):
-    """Fully-connected (s,a)-input continuous Q-function.
+    """Fully-connected + BN (s,a)-input Q-function with late action input.
 
     Actions are not included until the second hidden layer and not normalized.
     This architecture is used in the DDPG paper:
     http://arxiv.org/abs/1509.02971
 
     Args:
-        n_dim_obs: number of dimensions of observation space
-        n_dim_action: number of dimensions of action space
-        n_hidden_channels: number of hidden channels
-        n_hidden_layers: number of hidden layers
+        n_dim_obs (int): Number of dimensions of observation space.
+        n_dim_action (int): Number of dimensions of action space.
+        n_hidden_channels (int): Number of hidden channels.
+        n_hidden_layers (int): Number of hidden layers.
+        normalize_input (bool): If set to True, Batch Normalization is applied
+        nonlinearity (callable): Nonlinearity between layers. It must accept a
+            Variable as an argument and return a Variable with the same shape.
+            Nonlinearities with learnable parameters such as PReLU are not
+            supported.
+        last_wscale (float): Scale of weight initialization of the last layer.
+
     """
 
     def __init__(self, n_dim_obs, n_dim_action, n_hidden_channels,
-                 n_hidden_layers, normalize_input=True):
+                 n_hidden_layers, normalize_input=True,
+                 nonlinearity=F.relu, last_wscale=1.):
         self.n_input_channels = n_dim_obs + n_dim_action
         self.n_hidden_layers = n_hidden_layers
         self.n_hidden_channels = n_hidden_channels
         self.normalize_input = normalize_input
+        self.nonlinearity = nonlinearity
 
         super().__init__()
         with self.init_scope():
```

Comment:
Could you add a comment that `nonlinearity` does not need to be passed to `MLPBN` because `hidden_sizes` is empty?